modules/form-field/css/stories/stories_TextArea.tsx

```diff
@@ -29,7 +31,9 @@ storiesOf('CSS/Form Field/TextArea/Top Label', module)
             Label
           </label>
           <div className="wdc-form-field">
-            <textarea id="textarea" className="wdc-form-textarea" placeholder="Placeholder" />
+            <div className="wdc-form-textarea">
+              <textarea id="textarea" placeholder="Placeholder" />
```

Comment:
Multiple examples use the same `id`. Should we use different ids?

Reply:
I think it's fine, because they're not displayed simultaneously